src/librustc/hir/map/mod.rs

```diff
@@ -819,12 +784,6 @@ impl<'hir> Map<'hir> {
 
     /// Returns the `DefId` of `id`'s nearest module parent, or `id` itself if no
     /// module parent is in this map.
-    pub fn get_module_parent(&self, id: NodeId) -> DefId {
-        let hir_id = self.node_to_hir_id(id);
-        self.get_module_parent_by_hir_id(hir_id)
-    }
-
-    // FIXME(@ljedrz): replace the `NodeId` variant.
     pub fn get_module_parent_by_hir_id(&self, id: HirId) -> DefId {
```

Comment:
Now that the `NodeId` variants are gone, perhaps remove the `_by_hir_id` suffixes for brevity since there's no need for disambiguation?

Reply:
It's my plan, I was just saving it for last :slightly_smiling_face:. Should I apply these cleanups to this PR?

Reply:
@ljedrz I'd do whatever you believe makes the diffs cleanest.